mods/cnc/hotkeys.yaml

```diff
@@ -1,24 +1,34 @@
+NextProductionTab: PAGEDOWN
+	Description: hotkey-description-NextProductionTab
+	Types: Production
+	Contexts: hotkey-context-player
+
+PreviousProductionTab: PAGEUP
+	Description: hotkey-description-PreviousProductionTab
+	Types: Production
+	Contexts: hotkey-context-player
+
 ProductionTypeBuilding: E
-	Description: Building Tab
+	Description: hotkey-description-ProductionTypeBuilding
```

Comment:
all translations keys should be lower case

Reply:
The latter part is the hotkey ID - `ProductionTypeBuilding` in this case. Do we still want to reformat it, even if it doesn't match any more?

Reply:
yes. Just lower case it, I don't think we need to turn names into kebab case

Reply:
Updated